tools/pipelines/templates/include-test-task.yml

```diff
@@ -15,8 +15,8 @@ parameters:
   default: false
 
 steps:
-# Test - With coverage
-- ${{ if and(parameters.testCoverage, startsWith(parameters.taskTestStep, 'ci:test')) }}:
+# Test - With coverage - Don't run with coverage if the task is a PR and the task is a tinylicious service and stress test. We don't include coverage for these tests in our baseline coverage stats, so we don't need to run them with coverage here too.
+- ${{ if and(parameters.testCoverage, startsWith(parameters.taskTestStep, 'ci:test'), or(ne(variables['Build.Reason'], 'PullRequest'), and(not(contains(parameters.taskTestStep, 'realsvc:tinylicious')), not(contains(parameters.taskTestStep, 'stress:tinylicious')))))}}:
```

Comment:
You can probably make this a variable which will let you use better formatting, and then the if check here can just look at the variable. E.g.

https://github.com/microsoft/FluidFramework/blob/b053eb67bd6791349e2ac277697a8b2628f13864/tools/pipelines/templates/include-vars.yml#L28-L43

Reply:
Note: template yml files cannot declare variables, so I think what we'd need to do to implement this is to put the whole complex condition in the "caller file" where it passes the `testCoverage` parameter, and here keep a simple `${{ if parameters.testCoverage }}`. I like the idea though.